### DISCUSSION TITLE
known Issues & limitations: admint.io pixel-to-pixel testing

### DISCUSSION THREAD

**Discussion by gargajit** [FifthTry/admint.io#91] [Ideas & RFCs]
I have added the list of known issues. @gsalunke Please list down any other known issues.

`Known Issues`:

1. Animation is not supported: Carousel slider and Advertisement Card on-hover 
2. Font difference for Carousel's left nav-icon and right nav-icon for start and end sliders respectively

`fastn limitations`:

1. `animation` cannot be applied on `fastn` static site
2. `opacity` is not available in `fastn`

**Comment by gsalunke** [comment]
- [ ]  Slider movement animation is not matching - `animation` can't be applied while working on `fastn` static site.
- [x] Slider navigation's foreground and the background color is not matching - `opacity` is not available in `fastn`
- [ ] HOW IT WORKS sliders cannot be moved from Step 1 to Step 2 using a mouse movement after holding left-click  - `animation` can't be applied while working on `fastn` static site.
- [x] HOW IT WORKS slider navigation foreground and the background color is not matching - `opacity` is not available in `fastn`
- [ ] Slider 2.1: text STEP 1, font is not perfectly matching - critical to match - we have matched until 0.5px
- [ ] Slider 2.3: text STEP 3 font not perfectly matching - critical to match - we have matched until 0.5px
- [ ] Slider 2.4: Text STEP 4 font is not perfectly matching - critical to match - we have matched until 0.5px
- [ ] Slider 2.5: Text STEP 5 font is not matching - critical to match - we have matched until 0.5px
- [ ] Slider 2.6: Text STEP 6 font is not perfectly matching - critical to match - we have matched until 0.5px
- [ ] advrt-card on-mouse hover animation is not matching - - `animation` can't be applied while working on `fastn` static site.
- [ ] https://admint.io/learn/advertising-web-3-0/ - due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/learn/brand-campaign/ -  due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/learn/metaverse-is-the-new-playground-for-advertisement-2/  -  due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/learn/brand-campaigns/lorem-ipsum-is-simply-dummy-text-2/ -  due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/#/terms-conditions -  due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/#/privacy-policy -  due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/learn/  -  due to FTD markup para and ul, li formatting body of posts can't px to px matched
- [ ] https://admint.io/#/pricing - `animation` can't be applied while working on `fastn` static site.
